src/scripts/main.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+const section = document.querySelector('.chess');
+
+document.addEventListener('DOMContentLoaded', () => {
+  const div = document.createElement('div');
+
+  section.append(div);
+  div.className = 'chess__field';
+
+  const colors = ['chess__white', 'chess__black'];
```

Comment:
1. Create separate constants like `WHITE_CELL_COLOR` and `BLACK_CELL_COLOR`
2. Move these constants to the top of the file or create a new one